packages/http-specs/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@typespec/http-specs",
+  "version": "0.37.2",
+  "description": "Spec scenarios and mock apis",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "watch": "tsc -p ./tsconfig.build.json --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "clean": "rimraf dist/ temp/",
+    "validate-scenarios": "spec validate-scenarios ./http",
+    "generate-scenarios-summary": "spec generate-scenarios-summary ./http",
+    "validate-mock-apis": "spec validate-mock-apis ./http",
+    "check-scenario-coverage": "spec check-coverage ./http",
+    "serve": "spec serve ./http",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "homepage": "https://github.com/microsoft/typespec#readme",
+  "dependencies": {
+    "@typespec/spec": "workspace:*",
+    "@typespec/spec-api": "workspace:~",
+    "multer": "^1.4.5-lts.1",
+    "axios": "^1.7.5",
```

Comment:
wouldn't axios get pulled from the core package?

Reply:
Removed it